docs/source/asr/speaker_diarization/datasets.rst

```diff
@@ -1,72 +1,158 @@
+	
+``rttm_filepath`` (Optional):
+  
+  To evaluate a diarization system with known rttm files, one needs to provide Rich Transcription Time Marked (RTTM) files as ground truth label files. If RTTM files are provided, the diarization evaluation will be initiated. Here is one line from a RTTM file as an example:
 
 .. code-block:: bash
 
   SPEAKER TS3012d.Mix-Headset 1 331.573 0.671 <NA> <NA> MTD046ID <NA> <NA>
 
+``text`` (Optional):
+
+  Ground truth transcription for diarization with ASR inference. Provide the ground truth transcription of the given audio file in string format
+
+.. code-block:: bash
+
+  {"text": "this is an example transcript"}
 
-oracle_vad_manifest [Optional but suggested for good performace. This reduces FPR and TPR]:
-  - To perform just oracle diarization, that is taking speech activity time stamps from groundtruth RTTMs or another VAD from wild instead of NeMo VAD output, ``diarizer`` expects an oracle manifest json file that contains paths to audio files with offset for start time and duration of segment.
+``uem_filepath`` (Optional):
 
-To prepare an oracle manifest file, use the script from ``scripts`` folder as shown below:
+  The UEM file is used for specifying the scoring regions to be evaluated in the given audio file.
+  UEMfile follows the following convention: ``<uniq-id> <channel ID> <start time> <end time>``. ``<channel ID>`` is set to 1.
+
+  Example lines of UEM file:
 
 .. code-block:: bash
+  
+    TS3012d.Mix-Headset 1 12.31 108.98
+    TS3012d.Mix-Headset 1 214.00 857.09
+
+``ctm_filepath`` (Optional):
+    
+  CTM file is used for the evaluation of word-level diarization results and word-timestamp alignment. CTM file follows the following convention: ``<uniq-id> <speaker ID> <word start time> <word end time> <word> <confidence>`` Since confidence is not required for evaluating diarization results, it can have any value. Note that the ``<speaker_id>`` should be exactly matched with speaker IDs in RTTM. 
 
-  python $NeMo/scripts/ --paths2rttm_files=<path2groundtruth_rttm_files.txt> --paths2audio_files=<paths2audio_files.txt> --manifest_file=<output_oracle_manifest_file.json>
+  Example lines of CTM file:
+
+.. code-block:: bash
+  
+   TS3012d.Mix-Headset MTD046ID 12.879 0.32 okay 0
+   TS3012d.Mix-Headset MTD046ID 13.203 0.24 yeah 0
 
-Here ``paths2audio_files`` and ``path2groundtruth_rttm_files`` are files containing paths to audio files as shown above.
+
+Evaluation on Benchmark Datasets
+--------------------------------
+
+The following instructions can help one to reproduce the expected diarization performance on two benchmark English dialogue datasets. The following results are evaluations based on 0.25 second collar without evaluating overlapped speech. The evaluation is based on oracle VAD results from RTTM files. Therefore, speaker error rate (SER) is equal to confusion error since oracle VAD has no miss detection or false alarm.
```

Comment:
Speaker error rate is identical to confusion error rate right? here I support you wanan say diarization error rate?

Reply:
This is a mistake. Thanks for pointing this out.